src/features/navbar/navbar.html

```diff
@@ -36,6 +36,14 @@
             Log in
         </md-button>
 
+        <md-button class="md-raised md-small"
```

Comment:
jest obok przycisku logowania i myślę że to jest dobre miejsce, ewentualnie można dać też do widoku logowania